MapPointMapper/Parser/Parser.swift

```diff
@@ -206,17 +214,17 @@ class Parser {
   }
 
   /**
-  :abstract: Attempt to determine if a given string is in `Well-Known Text` format (GeoString as its referred to internally)
+  _abstract_: Attempt to determine if a given string is in `Well-Known Text` format (GeoString as its referred to internally)
 
-  :discussion: This strips any leading & trailing white space before checking for the existance of word characters at the start of the string.
+  _discussion_: This strips any leading & trailing white space before checking for the existance of word characters at the start of the string.
 
-  :param: input String to attempt determine if is in `Well-Known Text` format
+  - parameter input: String to attempt determine if is in `Well-Known Text` format
 
-  :returns: `true` if it thinks it is, `false` otherwise
+  - returns: `true` if it thinks it is, `false` otherwise
   */
   internal func isProbablyGeoString(input: String) -> Bool {
     let stripped = input.stringByStrippingLeadingAndTrailingWhiteSpace()
-    if let geoString = stripped.rangeOfString("^\\w+", options: .RegularExpressionSearch) {
+    if let _ = stripped.rangeOfString("^\\w+", options: .RegularExpressionSearch) {
```

Comment:
So this unused var warning annoyed me, but I ended up leaving it in because I think it provides a human-readable means of understanding what we're checking in this.  Either way, if we ain't using it why not
```swift
    if (stripped.rangeOfString("^\\w+", options: .RegularExpressionSearch) != nil) {
      return true
    }
```